lib/src/base-styles/_nx-card.scss

```diff
@@ -107,75 +78,64 @@
   }
 
   .nx-card__content {
-    -ms-grid-row: 2;
-    -ms-grid-column: 2;
-    grid-area: card-text;
+    align-self: center;
```

Comment:
What is the expected behavior when there is a header, content, and a callout large enough that there is extra vertical space amidst the header and content? Is this correct/acceptable:
![image](https://user-images.githubusercontent.com/3630091/107783285-15f5b900-6d18-11eb-8bf9-9b8db44a6ea8.png)

Reply:
> What is the expected behavior when there is a header, content, and a callout large enough that there is extra vertical space amidst the header and content? Is this correct/acceptable:

My expectation would be that if a designer specifies something like this that they will also design the rest of the layout appropriately to their needs (i.e. move the content text upwards, change its size, or whatever). I don't want to get too caught up in details of what happens to the example layouts in various scenarios because I think the example layouts are not prescriptive - as in "here are some examples of what you can do and how it's done", not "here are the layouts that we have made available".